README.md

```diff
@@ -586,6 +586,111 @@ crud.len('customers')
 ...
 ```
 
+### Statistics
+
+`crud` routers provide statistics on called operations.
+```lua
+-- Returns table with statistics information.
+crud.stats()
+```
+
+Statistics collect enabled by default on all routers.
+It can be disabled and re-enabled later.
```

Comment:
Is it possible to reset it?

Reply:
There is no explicit handle, but it looks looks like a relevant request

Reply:
Added reset handle